requirements/edx/github.txt

```diff
@@ -90,7 +90,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 -e git+https://github.com/edx/edx-reverification-block.git@0.0.5#egg=edx-reverification-block==0.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
 git+https://github.com/edx/xblock-lti-consumer.git@v1.0.9#egg=xblock-lti-consumer==1.0.9
-git+https://github.com/edx/edx-proctoring.git@0.12.21#egg=edx-proctoring==0.12.21
+-e git+https://github.com/amir-qayyum-khan/edx-proctoring.git@release#egg=edx-proctoring
```

Comment:
right now code is in `jia/hide-progress`, will you merge it in release first and then i will deploy edx platform to use release changes?